### ISSUE TITLE
Check for multiple blank lines?

### ISSUE THREAD

**Issue by bravo-kernel** [cakephp/cakephp-codesniffer#182]
Is there a reason there is no sniff for multiple blank lines? It happens to me all the time that I accidentally commit stuff like this:

```php
public function myTest()
{
    debug('a thing);


    debug('another thing')
}
```

**Comment by dereuromark** [comment]
No, just no one did it so far.
You can either write one, or copy [this](https://github.com/spryker/code-sniffer/blob/master/Spryker/Sniffs/Whitespace/EmptyLinesSniff.php) existing one for example into cakephp.

**Comment by ADmad** [comment]
The standards included in phpcs itself might have a sniff for it too. So just check that first so only updating our ruleset would be required.

**Comment by bravo-kernel** [comment]
Thanks for clarifying gents, I will not be able to look into this soon so consider this a placeholder, close if you like.

**Comment by bravo-kernel** [comment]
I will check my setup, could be that I f** up a version or something. Will report back

**Comment by ADmad** [comment]
The exact sniff responsible for checking multiple blank lines is [Squiz.WhiteSpace.SuperfluousWhitespace.EmptyLines](https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/PSR2/ruleset.xml#L50) which has been turned off in PSR2. But even turn that back on doesn't show errors, so not sure what's the issue.

**Comment by cleptric** [comment]
I was able to fix it and updated the PR #184 with now passing tests

**Comment by gmponos** [comment]
> The exact sniff responsible for checking multiple blank lines is Squiz.WhiteSpace.SuperfluousWhitespace.EmptyLines which has been turned off in PSR2. But even turn that back on doesn't show errors, so not sure what's the issue.

How did you turned it on? When I turn it on at my custom sniff. It works for.

This is how I do it:
```
    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace.EmptyLines">
        <severity>10</severity>
    </rule>
```

**Comment by cleptric** [comment]
@gmponos By adding 

```
<rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
 <properties>
  <property name="ignoreBlankLines" value="false"/>
 </properties>
</rule>
<rule ref="Squiz.WhiteSpace.SuperfluousWhitespace.EmptyLines">
 <severity>5</severity>
</rule>
```

As the PSR ruleset.xml sets `ignoreBlankLines` to `true` and `Squiz.WhiteSpace.SuperfluousWhitespace.EmptyLines severity` to 0.

**Comment by bravo-kernel** [comment]
Awesome job, thanks 👏
